src/Microsoft.AspNetCore.Authentication/RemoteAuthenticationOptions.cs

```diff
@@ -56,6 +56,11 @@ public string DisplayName
         /// </summary>
         public bool SaveTokensAsClaims { get; set; }
 
+        /// <summary>
+        /// Gets or sets the lifetime of the correlation cookie used to prevent cross-site scripting attacks.
```

Comment:
What was the name you suggested?  Maybe something generic like RemoteAuthenticationTimeout then?